### PR TITLE
Chore update appwrite version

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

- Update Appwrite version
- Update Changelog

## TODO
- [x] Update console
- [x] Add console release to changelog
 
## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+- Support for the all new bun runtime [#6230](https://github.com/appwrite/appwrite/pull/6230)
+
+## Fixes
+- Add _APP_OPENSSL_KEY_V1 to the compose file [#6199](https://github.com/appwrite/appwrite/pull/6199)
+- Fix: V2 functions env vars [#6215](https://github.com/appwrite/appwrite/pull/6215)
+- Fix: Don't update User Accessed At for Users and Teams APIs [#6222](https://github.com/appwrite/appwrite/pull/6222)
+- Fix: Git deploys with S3 [#6227](https://github.com/appwrite/appwrite/pull/6227)
+- Fix: manual internal id insertion [#6232](https://github.com/appwrite/appwrite/pull/6232)
+- Fix: function timeout [#6235](https://github.com/appwrite/appwrite/pull/6235)
+
```
Comment (abnegate) [inline CHANGES.md:13]:
```suggestion
- Fix: Missing keys when updating document via relationship [#320](https://github.com/utopia-php/database/pull/320)

```

File: app/init.php [hunk 2]
```diff
@@ -109,7 +109,7 @@
 const APP_USER_ACCCESS = 24 * 60 * 60; // 24 hours
 const APP_CACHE_UPDATE = 24 * 60 * 60; // 24 hours
 const APP_CACHE_BUSTER = 509;
```
Comment (abnegate) [inline app/init.php:111]: We need to increment this too

Comment (christyjacob4) [reply app/init.php]: done 👌

File: CHANGES.md (hunk 1 above)
Comment (stnguyen90) [inline CHANGES.md:14]:
```suggestion
- Ensure collections with datetime attributes are migrated [#6175](https://github.com/appwrite/appwrite/issues/6176)
- Ensure all user data are migrated including status, preferences, and labels [#6174](https://github.com/appwrite/appwrite/issues/6174)
- Allow disabling Custom SMTP Settings [#6203](https://github.com/appwrite/appwrite/issues/6203)
- Fix users pagination in Console [#6178](https://github.com/appwrite/appwrite/issues/6178)
- Fix team memberships migration [#6175](https://github.com/appwrite/appwrite/issues/6175]
- Fix events validation on create/update webhooks [#6218](https://github.com/appwrite/appwrite/issues/6218)
- Fix v2 function execution vars [#6209](https://github.com/appwrite/appwrite/issues/6209)
- Fix create attribute modal null when selecting same time twice [https://github.com/appwrite/console/pull/549](https://github.com/appwrite/console/pull/549)

```

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
+# Version 1.4.3
+
+## Features
+- Support for the all new bun runtime [#6230](https://github.com/appwrite/appwrite/pull/6230)
+
```
Comment (stnguyen90) [inline CHANGES.md:5]:
```suggestion
- Show a warning if GitHub app not configured in any of the Functions screens[#6180](https://github.com/appwrite/appwrite/issues/6180)

```

Comment (stnguyen90) [reply CHANGES.md:5]: we'll be showing this in a separate console seciton

File: CHANGES.md (hunk 1 above)
Comment (stnguyen90) [reply CHANGES.md:14]: Updated my previous comment to:

1. remove console related things
2. remove a duplicate entry
3. point to PRs

Comment (christyjacob4) [reply CHANGES.md:14]: Lets use this PR https://github.com/appwrite/appwrite/pull/6219 instead of https://github.com/appwrite/appwrite/issues/6218

Comment (christyjacob4) [reply CHANGES.md:18]: Updated 👌

**Review by abnegate — APPROVED** [review]

**Pull request merged**